doc/config.md

```diff
@@ -0,0 +1,211 @@
+[//]: # (This file is built out of deploy/doc/config.md, please do not edit it manually)  
```

Comment:
What's the difference between this file and `deploy/doc/config.md` ?

Reply:
good question. It's indeed not clear since the comment is in the deploy/doc/config.md.

I'll move the comment to Makefile in `doc/config.md` target.

The answer to your question: deploy/doc/config.md and deploy/doc/graphite-clickhouse.conf are sources for doc/config.md